mailbox/store/src/test/java/org/apache/james/mailbox/store/search/CombinedComparatorTest.java

```diff
@@ -123,12 +125,18 @@ public void createShouldConvertDisplayFrom() {
     @Test
     public void createShouldConvertId() {
         assertThat(CombinedComparator.create(ImmutableList.of(new Sort(SortClause.Id))).getComparators())
-            .containsOnly(MessageIdComparator.MESSAGE_ID_COMPARATOR);
+            .containsOnly(MessageComparators.MESSAGE_ID_COMPARATOR);
     }
 
     @Test
     public void createShouldReverse() {
-        assertThat(CombinedComparator.create(ImmutableList.of(new Sort(SortClause.DisplayFrom, Order.REVERSE))).getComparators())
-            .containsOnly(new ReverseComparator(HeaderDisplayComparator.FROM_COMPARATOR));
+        MailboxMessage message1 = mock(MailboxMessage.class);
+        when(message1.getUid()).thenReturn(MessageUid.of(1));
+        MailboxMessage message2 = mock(MailboxMessage.class);
+        when(message2.getUid()).thenReturn(MessageUid.of(2));
+
+        Comparator<MailboxMessage> comparator = CombinedComparator.create(ImmutableList.of(new Sort(SortClause.Uid, Order.REVERSE)));
+
+        assertThat(comparator.compare(message1, message2)).isGreaterThan(0);
```

Comment:
why this change?

Reply:
Because we can no longer assert that we are using a reversed comparator. The best way to check that it is reversed is actually to execute it.